portal/src/components/roles-and-groups/RolesAndGroupsEmptyView.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import cn from "classnames";
+import { Text } from "@fluentui/react";
+import styles from "./RolesAndGroupsEmptyView.module.css";
+import PrimaryButton from "../../PrimaryButton";
+
+const CreateButton: React.VFC<{
+  className?: string;
+  href?: string;
+  text: React.ReactNode;
+}> = function CreateButton(props) {
+  const { className, href, text } = props;
+  return (
+    <PrimaryButton
+      href={href}
+      className={className}
+      text={text}
+      iconProps={{ iconName: "Add" }}
+    />
```

Comment:
I thought we don't want to use `href` to avoid page from full reload?

Reply:
We can have href, but we need to handle the onClick event and preventDefault to prevent the page reload. You can see implementation of `ReactRouterLink.CustomComponent` to know about how I did it.